src/Evolution/Systems/ScalarAdvection/BoundaryConditions/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+set(LIBRARY ScalarAdvection)
```

Comment:
You can delete this line, as well as the one in `BoundaryCorrections/CMakeLists.txt`, because the `LIBRARY` variable is carried over from the parent scope. I haven't made up my mind yet if I like this pattern or not, but it's what is done in the rest of the code :)